R/mod_reportPreview.R

```diff
@@ -456,7 +507,7 @@ reportPreviewServer <- function(id, selected_pkg, maint_metrics, com_metrics,
       
       tagList(
         h5(code('{riskmetric}'), 'Assessment Date:'), selected_pkg$date_added(),
-        if('Risk Score' %in% report_includes()) tagList(h5('Risk Score:'), selected_pkg$score()) else "",
+        if('Risk Score' %in% report_includes()) tagList(hr(), br(), h5('Risk Score:'), selected_pkg$score()) else "",
```

Comment:
Why the change?

Reply:
I thought it needed a horizontal rule.  reverted.